[2023-10-11T19:27:21.860+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [queued]>
[2023-10-11T19:27:21.876+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [queued]>
[2023-10-11T19:27:21.876+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2023-10-11T19:27:21.893+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): trusted> on 2023-10-09 00:00:00+00:00
[2023-10-11T19:27:21.900+0000] {standard_task_runner.py:57} INFO - Started process 234 to run task
[2023-10-11T19:27:21.908+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'wikipedia_etl', 'trusted', 'scheduled__2023-10-09T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/wikipedia_etl.py', '--cfg-path', '/tmp/tmpvsm1t64s']
[2023-10-11T19:27:21.912+0000] {standard_task_runner.py:85} INFO - Job 17: Subtask trusted
[2023-10-11T19:27:21.974+0000] {task_command.py:415} INFO - Running <TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [running]> on host f711e14c143e
[2023-10-11T19:27:22.224+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='wikipedia_etl' AIRFLOW_CTX_TASK_ID='trusted' AIRFLOW_CTX_EXECUTION_DATE='2023-10-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-09T00:00:00+00:00'
[2023-10-11T19:27:35.463+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/wikipedia_etl.py", line 133, in trusted
    sc = SparkContext(conf=conf)
  File "/home/airflow/.local/lib/python3.8/site-packages/pyspark/context.py", line 201, in __init__
    SparkContext._ensure_initialized(self, gateway=gateway, conf=conf)
  File "/home/airflow/.local/lib/python3.8/site-packages/pyspark/context.py", line 436, in _ensure_initialized
    SparkContext._gateway = gateway or launch_gateway(conf)
  File "/home/airflow/.local/lib/python3.8/site-packages/pyspark/java_gateway.py", line 107, in launch_gateway
    raise PySparkRuntimeError(
pyspark.errors.exceptions.base.PySparkRuntimeError: [JAVA_GATEWAY_EXITED] Java gateway process exited before sending its port number.
[2023-10-11T19:27:35.485+0000] {taskinstance.py:1398} INFO - Marking task as UP_FOR_RETRY. dag_id=wikipedia_etl, task_id=trusted, execution_date=20231009T000000, start_date=20231011T192721, end_date=20231011T192735
[2023-10-11T19:27:35.601+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 17 for task trusted ([JAVA_GATEWAY_EXITED] Java gateway process exited before sending its port number.; 234)
[2023-10-11T19:27:35.696+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-11T19:27:35.974+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-11T19:35:26.672+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [queued]>
[2023-10-11T19:35:26.679+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [queued]>
[2023-10-11T19:35:26.679+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2023-10-11T19:35:26.692+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): trusted> on 2023-10-09 00:00:00+00:00
[2023-10-11T19:35:26.697+0000] {standard_task_runner.py:57} INFO - Started process 2923 to run task
[2023-10-11T19:35:26.699+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'wikipedia_etl', 'trusted', 'scheduled__2023-10-09T00:00:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/wikipedia_etl.py', '--cfg-path', '/tmp/tmpyjjy08ju']
[2023-10-11T19:35:26.702+0000] {standard_task_runner.py:85} INFO - Job 40: Subtask trusted
[2023-10-11T19:35:26.744+0000] {task_command.py:415} INFO - Running <TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [running]> on host f711e14c143e
[2023-10-11T19:35:26.811+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='wikipedia_etl' AIRFLOW_CTX_TASK_ID='trusted' AIRFLOW_CTX_EXECUTION_DATE='2023-10-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-09T00:00:00+00:00'
[2023-10-11T19:35:54.563+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-11T19:35:54.575+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=wikipedia_etl, task_id=trusted, execution_date=20231009T000000, start_date=20231011T193526, end_date=20231011T193554
[2023-10-11T19:35:54.611+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-11T19:35:54.625+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-11T19:51:13.237+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [queued]>
[2023-10-11T19:51:13.243+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [queued]>
[2023-10-11T19:51:13.243+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2023-10-11T19:51:13.254+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): trusted> on 2023-10-09 00:00:00+00:00
[2023-10-11T19:51:13.259+0000] {standard_task_runner.py:57} INFO - Started process 5667 to run task
[2023-10-11T19:51:13.261+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'wikipedia_etl', 'trusted', 'scheduled__2023-10-09T00:00:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/wikipedia_etl.py', '--cfg-path', '/tmp/tmpdwttd83e']
[2023-10-11T19:51:13.263+0000] {standard_task_runner.py:85} INFO - Job 60: Subtask trusted
[2023-10-11T19:51:13.300+0000] {task_command.py:415} INFO - Running <TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [running]> on host f711e14c143e
[2023-10-11T19:51:13.360+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='wikipedia_etl' AIRFLOW_CTX_TASK_ID='trusted' AIRFLOW_CTX_EXECUTION_DATE='2023-10-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-09T00:00:00+00:00'
[2023-10-11T19:51:42.741+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-11T19:51:42.752+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=wikipedia_etl, task_id=trusted, execution_date=20231009T000000, start_date=20231011T195113, end_date=20231011T195142
[2023-10-11T19:51:42.794+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-11T19:51:42.805+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-13T17:23:00.139+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [queued]>
[2023-10-13T17:23:00.145+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [queued]>
[2023-10-13T17:23:00.146+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2023-10-13T17:23:00.155+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): trusted> on 2023-10-09 00:00:00+00:00
[2023-10-13T17:23:00.161+0000] {standard_task_runner.py:57} INFO - Started process 2072 to run task
[2023-10-13T17:23:00.163+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'wikipedia_etl', 'trusted', 'scheduled__2023-10-09T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/wikipedia_etl.py', '--cfg-path', '/tmp/tmpehce97ho']
[2023-10-13T17:23:00.166+0000] {standard_task_runner.py:85} INFO - Job 20: Subtask trusted
[2023-10-13T17:23:00.203+0000] {task_command.py:415} INFO - Running <TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [running]> on host feba906f3341
[2023-10-13T17:23:00.264+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='wikipedia_etl' AIRFLOW_CTX_TASK_ID='trusted' AIRFLOW_CTX_EXECUTION_DATE='2023-10-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-09T00:00:00+00:00'
[2023-10-13T17:23:38.638+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-13T17:23:38.650+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=wikipedia_etl, task_id=trusted, execution_date=20231009T000000, start_date=20231013T172300, end_date=20231013T172338
[2023-10-13T17:23:38.675+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-13T17:23:38.686+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-16T11:30:17.678+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [queued]>
[2023-10-16T11:30:17.684+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [queued]>
[2023-10-16T11:30:17.685+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2023-10-16T11:30:17.696+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): trusted> on 2023-10-09 00:00:00+00:00
[2023-10-16T11:30:17.702+0000] {standard_task_runner.py:57} INFO - Started process 2059 to run task
[2023-10-16T11:30:17.704+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'wikipedia_etl', 'trusted', 'scheduled__2023-10-09T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/wikipedia_etl.py', '--cfg-path', '/tmp/tmpvn03s85c']
[2023-10-16T11:30:17.706+0000] {standard_task_runner.py:85} INFO - Job 20: Subtask trusted
[2023-10-16T11:30:17.745+0000] {task_command.py:415} INFO - Running <TaskInstance: wikipedia_etl.trusted scheduled__2023-10-09T00:00:00+00:00 [running]> on host ab44499fca46
[2023-10-16T11:30:17.808+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='wikipedia_etl' AIRFLOW_CTX_TASK_ID='trusted' AIRFLOW_CTX_EXECUTION_DATE='2023-10-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-09T00:00:00+00:00'
[2023-10-16T11:30:48.171+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-16T11:30:48.180+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=wikipedia_etl, task_id=trusted, execution_date=20231009T000000, start_date=20231016T113017, end_date=20231016T113048
[2023-10-16T11:30:48.228+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-16T11:30:48.239+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
